lesson_summary/python/comments.md

```diff
@@ -0,0 +1,37 @@
+ ###Lesson Description - Using Comments
+
+When writing scripts, we often want to leave ourselves notes or explanations. 
+Python (along with most scripting languages) uses the # character to signify
+that the line should be ignored and not executed.
+
+
+
+Single Line Comment
+
+We can comment out a whole line:
+
+  Eg. 1  # This is a full like comment
```

Comment:
Can add the back ticks for Eg. 1 and Eg. 2? Thanks!
For inline code you can use the back ticks like this: 

\` Eg. 1 # This is a full like comment` 

so it shows as 

` Eg. 1 # This is a full like comment`